tests/test_views.py

```diff
@@ -1013,6 +1015,20 @@ def _setup_enrollment_client(self, client_mock):
         client.get_course_modes.return_value = self.dummy_demo_course_modes
         client.get_course_enrollment.return_value = None
 
+    def _setup_ecommerce_client(self, client_mock):
```

Comment:
@itsjeyd Isn't this the same case as: https://github.com/edx/edx-enterprise/pull/110#discussion_r121084292?